docs/release-notes/release-notes-0.7.2.md

```diff
@@ -0,0 +1,85 @@
+Joinmarket-clientserver 0.7.2:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.7.2>
+
+This release requires Python 3.6+ - see "Upgrading" section in the 0.7.0 release notes for more information if needed.
+
+This is mostly bugfix release, fixing some issues with certain cases of the new Payjoin functionality; anyone wishing to use that functionality should upgrade, to make it work much more reliably (remember: Payjoin falls back to normal payment if anything goes wrong).
+It also adds a bit more functionality, in particular QR codes generate-able by the Payjoin receiver.
+
+It also bugfixes the 'not-self' broadcast feature introduced in the last release.
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading 
+=========
+
+To upgrade:
+
+(If you are upgrading from a version pre-0.6.2 then please see the section "Move user data to home directory" in [the 0.6.2 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.6.2.md), and follow the instructions there (except the `commitmentlist` file - that can actually be left alone, the previous release notes were wrong on this point).)
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.7.0.md).)
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Notable changes
+===============
+
+### BIP 78 (Payjoin) improvements and fixes
+
+See the [previous](release-notes-0.7.1.md) for details on how to use Payjoin as a receiver, or for more details on sending and receiving, see [this](../PAYJOIN.md) documentation.
+In this release, a couple extra elements of functionality are added: the receiver can now create a QR code for the sender, in the Payjoin receiver dialog. Also, the tor configuration settings: `tor_control_port` and `tor_control_host` are added in the `[PAYJOIN]` section of `joinmarket.cfg` for people who want to use non-default configurations of Tor.
```

Comment:
```suggestion
In this release, a couple extra elements of functionality are added: the receiver can now create a QR code for the sender, in the Payjoin receiver dialog. Also, the Tor configuration settings: `tor_control_port` and `tor_control_host` are added in the `[PAYJOIN]` section of `joinmarket.cfg` for people who want to use non-default configurations of Tor.
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/db7dd77403d1d5d56c348d62f4fd75007c00867d